is_core/auth_token/middleware.py

```diff
@@ -33,10 +33,12 @@ def process_response(self, request, response):
         # Save the session data and refresh the client cookie.
         # Skip session save for 500 responses, refs #3881.
         if response.status_code != 500 and hasattr(request, 'token') and request.token.is_active:
-            if not request.token.expiration:
```

Comment:
Myslim si, ze ta podminka byla spatne. Expirace tokenu (Token.is_expired) totiz nesouhlasila s expiraci cookie. Bez "**not**" jiz tato logika plati.
Dava to i smysl - kdyz nemam na tokenu nastavenou expiraci, cookie take nastavim bez expirace (max_age = None, expires = None). Kdyz ji token nastavenou ma, nastavim ji i na cookie.